README.md

```diff
@@ -256,19 +256,20 @@ and limitations for this exporter.
+| `generatr.markdown.flexmark.extensions` | Additional extensions to the markdown generator to add new markdown capabilities. [More Details](https://avisi-cloud.github.io/structurizr-site-generatr/main/extended-markdown-features/)                                                                                                                                                       | Tables                          | `Tables,Admonition`                                  |
+| `generatr.svglink.target`               | Specifies the link target for element links in the exported svg                                                                                                                                                                                                                                                                                  | `_top`                          | `_self`                                              |
+| `generatr.site.exporter`                | Specifies the UML exporter, can be `c4` (uses the `C4PlantUMLExporter`) or `structurizr` (uses the `StructurizrPlantUMLExporter`)                                                                                                                                                                                                                | `c4`                            | `structurizr`                                        |
+| `generatr.site.externalTag`             | Software systems containing this tag will be considered external                                                                                                                                                                                                                                                                                 |                                 |                                                      |
+| `generatr.site.nestGroups`              | Will show software systems in the left side navigator in collapsable groups                                                                                                                                                                                                                                                                      | `false`                         | `true`                                               |
+| `generatr.site.views.cdn`               | Specifies what the CDN base url will be.                                                                                                                                                                                                                                                                                                         | `https://cdn.jsdelivr.net/npm/` | `https://www.unpkg.com/react@16.7.0/umd/`            |
```

Comment:
A better configuration name would be `generatr.site.cdn`. Is it possible to use `https://www.unpkg.com/` as example? Let's do this though only when this works, otherwise lets use a pseudo internal url like `https://cdn.my-company/npm/``

Reply:
"Specifies what the CDN base url will be." -> "Specifies the CDN base location for fetching NPM packages for browser runtime dependencies. Defaults to `jsDelivr`, but can be changed to e.g. an on-premise location."

Reply:
- Configuration name changed
- https://www.unpkg.com/ leads to the main website of unpkg, my preference would be using a pseudo internal url
- Description of generatr.site.cdn changed